src/exchange/account/delegation/account-delegation.controller.js

```diff
@@ -175,7 +175,7 @@ angular.module('Module.exchange.controllers').controller(
       this.services.messaging.resetMessages();
       this.loading = true;
       // filter by domai name or free text search
-      let filter = this.searchValue || _.get(this.selectedDomain, 'displayName');
+      const filter = this.searchValue || _.get(this.selectedDomain, 'displayName');
```

Comment:
Do you have eslint installed in your IDE ? It should tell you when to use `const` and when to use `let`